packages/backend-output-schemas/src/auth/v1.ts

```diff
@@ -13,6 +13,8 @@ export const authOutputSchema = z.object({
     facebookClientId: z.optional(z.string()),
     googleClientId: z.optional(z.string()),
 
+    allowUnauthenticatedIdentities: z.string(), // boolean as string 'true' | 'false'
```

Comment:
Is this intended to be a required property? As written, new versions of backend-output will no longer be able to parse existing stack metadata because this field will not be present

Reply:
updated to be optional 👍